### PR TITLE
Add instructions to README, and update link on Get Started page

### PR DESCRIPTION
**Pull request by blaesy**
Added instructions in README with an anchor (#).
Added redirect from Get started page to the anchor in README in the original repo.

Issue #122

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,10 +35,21 @@ All commands are run from the root of the project, from a terminal:
 | `npm run astro ...`    | Run CLI commands like `astro add`, `astro check` |
 | `npm run astro --help` | Get help using the Astro CLI                     |
 
+## Instructions on how to run your own server
+The archive file contains a dist/ folder that needs to be placed into a copy of this cloned repo. Then you can serve it using grunt serve. On a production site, you should to use these environment variables:
```
Comment (taoeffect) [inline README.md:39]: Ahh, I see... what happened, sorry for not being clear in the issue - but this section should be added to the README in the [group-income repo](https://github.com/okTurtles/group-income), not the groupincome.org repo. Can you send in a PR there?

For this PR we just need the change below to link to the group-income repo.

(Also, can you add an empty line after the heading too?)

File: README.md [hunk 2]
```diff
@@ -35,10 +35,21 @@ All commands are run from the root of the project, from a terminal:
...
+
+
 ## 👀 Want to learn more?
 
 Check out [our documentation](https://docs.astro.build) or jump into our [Discord server](https://astro.build/chat).
 
 ## Credit
 
-This theme is based off of the lovely [Bear Blog](https://github.com/HermanMartinus/bearblog/).
+This theme is based off of the lovely [Bear Blog](https://github.com/HermanMartinus/bearblog/).
```
Comment (blaesy) [inline README.md:55]: No changes, Git marks as a change - not relevant.

File: src/pages/get-started.astro [hunk 3]
```diff
@@ -21,7 +21,7 @@ import Servers from '../components/Servers.vue';
       <Servers />
       <p class="c-container-para">
         Don’t see a server you like?
-        <a href="https://github.com/okTurtles/group-income" class="link server-link" target="_blank">
+        <a href="https://github.com/okTurtles/groupincome.org#-instructions-on-how-to-run-your-own-server" class="link server-link" target="_blank">
```
Comment (taoeffect) [inline src/pages/get-started.astro:24]: Please update this to point to the `group-income` repo's README, and double-check that the hash is correct as I'd be surprised if it started with a `-`...

**Review by taoeffect — CHANGES_REQUESTED** [review]
Awesome, thanks @blaesy!

Left a review :)

**Comment by taoeffect** [comment]
Stale

**Pull request closed without merging**